pkg/diagnose/connectivity/core_endpoint_test.go

```diff
@@ -59,3 +60,33 @@ func TestSendHTTPRequestToEndpoint(t *testing.T) {
 	assert.Equal(t, statusCode, 400)
 	assert.Equal(t, string(responseBody), "Bad Request")
 }
+
+func TestAcceptRedirection(t *testing.T) {
+	t.Run("TestRedirection", func(*testing.T) {
```

Comment:
Since we only have a single test in `TestAcceptRedirection`, using `t.Run` is not necessary.

Reply:
fixed, thx